assets/views/layout.ejs

```diff
@@ -25,7 +25,13 @@
     </script>
   </head>
 
-  <body ng-controller="userCtrl">
+  <body ng-controller="userCtrl" ng-cloak ng-init="init({<%=
+    (typeof user !== 'undefined' ? 'user:' + JSON.stringify(user) + ',' : '') +
+    (typeof refUser !== 'undefined' ? 'refUser:' + JSON.stringify(refUser) + ',' : '') +
+    (typeof flairs !== 'undefined' ? 'flairs:' + JSON.stringify(flairs) + ',' : '') +
+    (typeof flairApps !== 'undefined' ? 'flairApps:' + JSON.stringify(flairApps) + ',' : '') +
+    (typeof query !== 'undefined' ? 'query:' + JSON.stringify(query) + ',' : '')
+  %>})">
```

Comment:
Here's where the EJS variables get injected into angular variables. Essentially, the `$scope.init()` function is called with an object containing the existing EJS variables.